src/main/kotlin/graphics/scenery/DetachedHeadCamera.kt

```diff
@@ -106,47 +106,52 @@ class DetachedHeadCamera(@Transient var tracker: TrackerInput? = null) : Camera(
         this.name = "DetachedHeadCamera-${tracker ?: "${counter.getAndIncrement()}"}"
     }
 
-    override fun createSpatial(): CameraSpatial {
-        return object: CameraSpatial(this) {
-            override var projection: Matrix4f = Matrix4f().identity()
-                get() = if(tracker != null && tracker is Display && tracker?.initializedAndWorking() == true) {
-                    (tracker as? Display)?.getEyeProjection(0, nearPlaneDistance, farPlaneDistance) ?: super.projection
-                } else {
-                    super.projection
-                }
-                set(value) {
-                    super.projection = value
-                    field = value
-                }
-            /**
-             * Returns this camera's transformation matrix, taking an eventually existing [TrackerInput]
-             * into consideration as well.
-             */
-            override fun getTransformation(): Matrix4f {
-                val tr = Matrix4f().translate(this.position * (-1.0f))
+    override fun createSpatial(): CameraSpatial = DetachedHeadCameraSpatial(this)
```

Comment:
This is an unwanted code style change

Reply:
Nope, this is needed. Kryo cant sync anonymous classes